src/physics/mod_dust.t

```diff
@@ -463,9 +472,9 @@ contains
     logical, intent(in)             :: qsourcesplit
     logical, intent(inout)            :: active
 
-    double precision :: ptherm(ixI^S), vgas(ixI^S, 1:ndir)
+    double precision :: ptherm(ixI^S), vgas(ixI^S, 1:ndir), gravity_field(ixI^S, 1:ndir)
```

Comment:
gravity_field(ixI^S, 1:ndir) should be gravity_field(ixI^S, 1:ndim) or gravity_field(ixI^S, ndim)